assets/scripts/app.js

```diff
@@ -31,5 +32,11 @@ function init() {
     html.classList.add('is-loaded');
     html.classList.add('is-ready');
     html.classList.remove('is-loading');
-}
 
+    // Fontsfaces watcher
+    fontFacesWatcher([
+        { fontFamily: 'Webfont', style: 400, weight: 'normal' },
```

Comment:
Redundant prefix for family property.

```suggestion
        { family: 'Webfont', style: 400, weight: 'normal' },
```